### PR TITLE
Add a docid REPL to ApiPort client

### PR DESCRIPTION
**Pull request by twsouthwick**

### CODE REVIEW

File: docs/BreakingChanges/110 NetDataContractSerializer fails to deserialize a ConcurrentDictionary serialized with by a different .NET version.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-## 110: NetDataContractSerializer throws an exception while trying to de-serialize a ConcurrentDictionary serialized with by a different .NET version
+## 110: NetDataContractSerializer fails to deserialize a ConcurrentDictionary serialized with by a different .NET version
```
Comment (mjrousos) [inline docs/BreakingChanges/110 NetDataContractSerializer fails to deserialize a ConcurrentDictionary serialized with by a different .NET version.md]: May as well clean up the "with by" typo, even though it's not introduced by your change.

File: src/ApiPort/ApiPort.csproj [hunk 2]
```diff
@@ -34,7 +34,7 @@
     <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
   <PropertyGroup>
-	<IntermediateOutputPath>$(OutputIntermediate)\$(AssemblyName)</IntermediateOutputPath>
+    <IntermediateOutputPath>$(OutputIntermediate)\$(AssemblyName)</IntermediateOutputPath>
```
Comment (mjrousos) [inline src/ApiPort/ApiPort.csproj]: ?

**Comment by mjrousos** [comment]
LGTM

File: docs/BreakingChanges/110 NetDataContractSerializer fails to deserialize a ConcurrentDictionary serialized with by a different .NET version.md (hunk 1 above)
Comment (twsouthwick) [inline docs/BreakingChanges/110 NetDataContractSerializer fails to deserialize a ConcurrentDictionary serialized with by a different .NET version.md]: Good call

**Pull request merged**